### PR TITLE
Move some test code from imSim to here

### PR DESCRIPTION
**Pull request by jchiang87**
* The new code tests various methods of the objects.  I added `data/ci_sample/galaxy_9557.parquet` since the existing galaxy files don't have the `.ellipticity _*` columns needed by `GalaxyObject.get_gsobject_components(...)`.
* I also updated the other `test*.py` files to account for the recent API changes.

### CODE REVIEW

File: tests/test_pointsource.py
Comment (JoanneBogart) [inline tests/test_pointsource.py:1]: This works as is, but would fail if the calls to `explore` with object type `sn` were uncommented.  Up to you if you want to fix this now (by changing the `sn` to `sncosmo` in the various calls to `explore` and add the lines for that object type to `data/ci_sample/skyCatalog.yaml`) or we leave it for a separate PR.  There is more clean-up to be done on this file.

**Review by JoanneBogart — APPROVED** [review]
The file `test_pointsource.py` could use some work and `data/ci_sample/skyCatalog.yaml` should be adjusted, but that is somewhat separate from the main thrust of this PR.  It can be done now or in a separate PR.

File: tests/test_pointsource.py
Comment (jchiang87) [reply tests/test_pointsource.py:1]: I'd prefer that these fixes be addressed in a separate PR, perhaps where the `test_pointsource.py` code is revised to contain real unit tests.
